docs/tutorials/calibrating_armonk.ipynb

```diff
@@ -0,0 +1,1694 @@
4wcqa/XZcvoZO2sWqVnYPTt27FUcjpw0UX6/YknOovgn/+s3622TwcGD9bBexs2dF77Y9MmHZSanw8jRqSseLYT6tZftUp34EI58MB1nf5vaOh5TZR4MYKfRojAjTfqv2fO7Jj28uKL+iFywgl6ClC6UFSkHwytrZ3dg+nmzre47DLdxrNmdaSdff117Sbcd189IyFdyM7uaD/LhR/696hR6ePNAN0RLyuDzz7r8GB8+qn2XvXurT086YJIR2xN6H0b6rnJykp6sRwj1K2/cSMUFHT+/qijOs/ZKyiAAQNwBCP4acaECTqie9ky2t1C//ynfrdW3EonIo3jp1vAnsXQodr129zcYfk9/bR+Tyfr3iLSOH66jd9b5OTAuefqv594Qr8/+qh+P/fcHUXC60Qax09Hd77FmWfq9/XrIRDoPI4xcGDn6YeBgGr3CtiNEfw0IzMTfvMb/ffvfqfdoW+8oV3f1hrT6UToOL5S8O672nUG6WfhA1xxhX5/6CEd0PXii/r/dInODyU8AU9ra4copJvgQ4db/+mndaKdp57S/6eTO9/CEvyqKlgX9Ginwwp5XWEJ+GuvwVVXCRB57l1eXoCKCnFsJcQ0cpwYLC64AKZN04Frl1+u064edRSUl6e6ZPZjWfivvaanLVlz8nNyYMyY1JXLKU44QXtwvvlGu/gbGuDQQ729NkJXWIF7zz+vr91NmzrGt9NR8EeN0sMyS5bApZfqWIVRozp+h3TCqtPrr2v39bBhekwf0tPCHzZMd2QWLdLj+CCIKPLz9bhUYaG27CsqMpk+3blyGAs/DcnJgYoK/bcVwZ1O0fmh7LuvDu7asEGLff/+2sPx+efpFdRlkZGhO3HQMV3LcgWnG/vso8WgqUmPfYIe554wIX0i1kMRgQsv1H9bbfuzn6WuPE6y777whz/AMcdosVu+XI9xl5ToTIvpiOXWt9r2zjuFe+6BQYPg7rth7VphxgxnY1OM4Kcpl1yiH44W6Th+D7pzM3OmvpleeEH3nm+/PT1SzHbFxIk6khl0B8B6kKQbOTk6iO2TT3TUcnOz7tj94x+63unI+ed31C0vL307cyJw3XXw5pt62HHxYp1e9tVXvZ8KuytCx+VLSuDnP4dJk3S+/UmTcMyNH0qa3jaGwkL45S/136NGdazclI5cc42er3zGGTq6O90pKekQgmOP1V6NdKW0VFuD/funV+R2VwwcCOPG6b/Hj0+faaXdkZWlA2wvuSR9EkdFwnLrgx6OKyxMfhl8cAv5l2uv1W6y009PdUkMdjNtml5HwBq6MaQPv/+99uBMm5bqkhjs5t579SyMX/86Nec3gp/GFBRo97Yh/dh55474DEN6sf/+OlDRkH4cfrh+pQrj0jcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQcYwTcYDAaDwQeIUirVZXAMEdkAfGfzYcuAjTYf042YeqYXpp7pgx/qCKaeiTBEKVUe/mFaC74TiMiHSqmDUl0OpzH1TC9MPdMHP9QRTD2dwLj0DQaDwWDwAUbwDQaDwWDwAUbwY+fhVBcgSZh6phemnumDH+oIpp62Y8bwDQaDwWDwAcbCNxgMBoPBBxjBNxgMBoPBBxjBjxIRuVxElotIo4gsEpEjU10mOxGRG0VkoYhsFZENIvKyiOyT6nI5TbDeSkT+lOqy2I2IDBSRJ4Lt2SgiS0Xk6FSXy05EJFNEZoTcm8tF5DYRyUp12RJBRI4SkZdE5Pvg9Tkx7HsRkWkiskZEtotIlYjsnaLixk139RSRbBGZKSKfiEi9iKwVkadFZHAKixwXPbVn2Lb/F9ymwu5yGMGPAhE5C7gX+B0wCngPeNWLF143jAUeAA4DjgVagf+KSGkqC+UkInIocCnwSarLYjci0gd4FxDgFGAEcBWwPoXFcoIbgCuAq4G9gGuC/9+YykLZQBGwBF2f7RG+/zXwK3SbHoxu1zki0itpJbSH7upZABwI/Db4fjqwC/AfD3boempPAERkPHAIsMaRUiilzKuHF/AB8EjYZ18Dt6e6bA7WuQgIAKeluiwO1a83sAw4BqgC/pTqMtlcv98B76a6HEmo57+AJ8I+ewL4V6rLZmMd64CJIf8LsBa4KeSzfGAbcFmqy2tXPbvYZiSggH1TXV676wkMAb5Hd85XABV2n9tY+D0gIjnAaOD1sK9eR1vD6UovtAeoJtUFcYiHgdlKqbdSXRCHOAP4QESeFZH1IrJYRK4UEUl1wWzmHeAYEdkLQERGoj1Ur6S0VM4yDBhAyDNJKbUdmEt6P5MAioPvafVcCnos/g7cppT63KnzeM0tkgrKgExgXdjn64Djk1+cpHEvsBiYn+Jy2I6ITAJ2B85PdVkcZFfgcuBu4PfAAcB9we/SKV5hJrpzulREAuhn2m+VUg+ktliOMiD4HumZtFOSy5I0gsbXH4CXlVKrU10em7kV2KiUetDJkxjBN+yAiNwFHAEcoZQKpLo8diIiw9Hu7iOUUi2pLo+DZAAfKqWsseyPRWQP9Ph2Ogn+WcCFwLnAZ+iOzb0islwp9WgqC2awj6AF/FegD/Cj1JbGXkRkLDARfe06inHp98xG9Fh2/7DP+wPVyS+Os4jI3cA5wLFKqW9TXR4HGIP22nwmIq0i0gocDVwe/D83tcWzjbXA0rDPPgfSKdAU4E6gUin1jFLqU6XUU8BdeD9orzus545fnkmWu3s/4Dil1KYUF8luxgIDgbUhz6QhwEwRsdWTYQS/B5RSzcAi4ISwr05AR+unDSJyLx1i/0Wqy+MQ/wT2RfemrdeHwDPBv5tTUir7eRcYHvbZnti/XHSqKUB3yEMJkN7PtuVoYW9/JolIHnAk6fdMygaeRYv9MUqptOvQoGdH7UfnZ9Ia9HDccXaeyLj0o
```

Comment:
I would add a comment here to say that you are checking the results but the `Amplitude.update` method below does this math itself.

Reply:
See here: 267bc35